src/components/badges/BadgeBlock.tsx

```diff
@@ -64,12 +67,12 @@ function Badge({ proof }: { proof: BaseProof }) {
           <>
             {proof instanceof ERC721Proof ? (
```

Comment:
Maybe it's better to set `proof instanceof ERC721Proof` in the `const` variable. As you did with `isEmailProof`